config/default.yml

```diff
@@ -292,6 +292,7 @@ Layout/EmptyLinesAroundClassBody:
     - empty_lines_except_namespace
     - empty_lines_special
     - no_empty_lines
+    - beginning_only
```

Comment:
Should we add an `ending_only` to compliment this new configuration?

Reply:
I thought this formatting option was questionable, but for the sake of completeness it makes sense. It was also an easy addition.